### PR TITLE
Krishguptadev/feature-dynamic-blogs-3

### LINKED ISSUE #3: [FEATURE] Dynamic Blogs
### Detailed description

Dynamic Blogs using github actions

### Context

\-

### Possible implementation

Use the popular blog-post-workflow action

### Additional information

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by krishguptadev**
fixes #3

### CODE REVIEW

File: .github/workflows/blog-posts.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Latest blog post workflow
+on:
+  schedule: # Run workflow automatically
+    - cron: '0 * * * *' # Runs every hour, on the hour
+  workflow_dispatch: # Run workflow manually (without waiting for the cron to be called), through the Github Actions Workflow page directly
+
+jobs:
+  update-readme-with-blog:
+    name: Update this repo's README with latest blog posts
```
Comment (Panquesito7) [inline .github/workflows/blog-posts.yml:9]:
```suggestion
    name: Update the README with the latest blog posts
```

File: README.md [hunk 2]
```diff
@@ -1,9 +1,7 @@
-# Welcome to Blogging Category
+<!-- markdownlint-disable-next-line -->
+<div align="center"><p><a href="https://app.netlify.com/sites/devprotocol/deploys"><img src="https://img.shields.io/netlify/bddfebe4-8553-4de6-9ddb-522ce7f67842?style=flat-square&logo=netlify"></a> <a href="https://discord.gg/VwJp4KM"><img src="https://img.shields.io/discord/547215761341546497?style=flat-square&logo=discord"></a></p><br><img height="70px" src="https://github.com/web3community.png"><br><h1>Web3community Blog</h1><br><p><a href="https://dev.to/web3community">Visit blog!</a> · <a href="#">Request a article</a> · <a href="#">Write for us 🍴</a></p></div>
```
Comment (Panquesito7) [inline README.md:2]: This is the wrong Netlify link. Please fix it. 🙂

Comment (Panquesito7) [inline README.md:2]:
```suggestion
<div align="center"><p><a href="https://app.netlify.com/sites/devprotocol/deploys"><img src="https://img.shields.io/netlify/bddfebe4-8553-4de6-9ddb-522ce7f67842?style=flat-square&logo=netlify"></a> <a href="https://discord.gg/VwJp4KM"><img src="https://img.shields.io/discord/547215761341546497?style=flat-square&logo=discord"></a></p><br><img height="70px" src="https://github.com/web3community.png"><br><h1>Web3community Blog</h1><br><p><a href="https://dev.to/web3community">Visit blog</a> · <a href="#">Request a article</a> · <a href="#">Write for us 🍴</a></p></div>
```

Comment (krishguptadev) [reply README.md:2]:
```suggestion
<div align="center"><p><a href="https://discord.gg/VwJp4KM"><img src="https://img.shields.io/discord/547215761341546497?style=flat-square&logo=discord"></a></p><br><img height="70px" src="https://github.com/web3community.png"><br><h1>Web3community Blog</h1><br><p><a href="https://dev.to/web3community">Visit blog!</a> · <a href="#">Request a article</a> · <a href="#">Write for us 🍴</a></p></div>
```

File: .github/workflows/blog-posts.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+name: Latest blog post workflow
+on:
+  schedule: # Run workflow automatically
+    - cron: '0 * * * *' # Runs every hour, on the hour
+  workflow_dispatch: # Run workflow manually (without waiting for the cron to be called), through the Github Actions Workflow page directly
```
Comment (Panquesito7) [inline .github/workflows/blog-posts.yml:5]:
```suggestion
  workflow_dispatch: # Run workflow manually (without waiting for the cron to be called), through the GitHub Actions Workflow page directly
```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Let's wait for the other maintainers' reviews. 😄

**Review by vinzvinci — APPROVED** [review]

**Pull request merged**